snuba/datasets/processors/spans_processor.py

```diff
@@ -393,13 +238,11 @@ def process_message(
             # At time of writing, there are no reads of the values in the `processed`
             # dictionary to inform values in other functions.
             # Ideally we keep continue that rule
-            self._process_base_event_values(processed, event_dict, common_span_fields)
-            self._process_tags(processed, event_dict, common_span_fields)
-            self._process_measurements(processed, event_dict)
-            self._process_module_details(processed, event_dict)
+            self._process_span_event(processed, span_event)
+            self._process_tags(processed, span_event)
+            self._process_measurements(processed, span_event)
+            self._process_sentry_tags(processed, span_event)
             processed_rows.append(processed)
-            processed_spans = self._process_spans(event_dict, common_span_fields)
-            processed_rows.extend(processed_spans)
 
         except Exception as e:
```

Comment:
At some point we would need to get rid of this catch all exception? Can you add a ticket in JIRA to track it? When we go to GA then we would need to clean this up.